### DISCUSSION TITLE
是否可以考虑增加dockerfile进行部署？

### DISCUSSION THREAD

**Discussion by Gu-f** [bs-community/blessing-skin-server#365] [Ideas]
是否可以考虑增加dockerfile进行部署？这样可以很大程度上减轻部署和备份的难度。

**Comment by Big-Cake-jpg** [comment]
已经有了（特性测试）
请查看feature/docker-build分支

**Reply by Gu-f** [reply]
看到了，目前的docker-build分支是否可以用于生产环境的部署？

**Reply by Big-Cake-jpg** [reply]
可以，但是不建议，毕竟都标上了feature

**Reply by Gu-f** [reply]
好的，非常感谢。
